src/biogeochem/DryDepVelocity.F90

```diff
@@ -399,10 +404,14 @@ subroutine depvel_compute( bounds, &
                end if
             else if ( snow_depth(c) > 0 ) then
                index_season = 4
-            else if(elai(pi) > 0.5_r8*maxlai) then
+            else if(.not.use_fates .and. elai(pi) > 0.5_r8*maxlai) then
                index_season = 1
             endif
 
+            if(use_fates)then
+              index_season = canopystate_inst%drydep_season_patch(pi)              
+            endif
+
             if (index_season<0) then
```

Comment:
There is .not. use_fates above, however, it is not here.
This can possibly overwrite season index.